RELEASE.rst

```diff
@@ -0,0 +1,36 @@
+The SunPy project is very happy to announce the release of a new package "ndcube".
+
+ndcube is a package built to represent a wide variety of data, of any number of
+dimensions, along with coordinate information described by FITS-WCS. ndcube is
+based on the astropy NDData class, and adds functionality for slicing both the
+data and the WCS simultaneously, plotting with matplotlib and support for extra
+coordinate information along any of the axes not described by WCS.
```

Comment:
Since `NDCubeSequence` is mentioned in the next paragraph, `NDCube` should be mentioned in this one.